src/core.js

```diff
@@ -1819,20 +1894,13 @@ define([
             mugs = multiselect ? mug : [mug],
             $baseToolbar = $(question_toolbar({
                 comment: multiselect ? '' : mug.p.comment,
-                isDeleteable: mugs && mugs.length && _.every(mugs, function (mug) {
-                    return _this.isMugRemoveable(mug, mug.hashtagPath);
-                }),
                 isCopyable: !multiselect && mug.options.isCopyable,
             }));
-        $baseToolbar.find('.fd-button-remove').click(function () {
-            var mugs = _this.getCurrentlySelectedMug(true, true);
-            form.removeMugsFromForm(mugs);
-            _this.refreshCurrentMug();
-        });
```

Comment:
Oh, if it's in the multiple-questions-selected view then I think that's fine since it's still possible to delete multiple. We could also enable the *Cut* command...